tools/checkpoint_conversion/convert_bloom_checkpoints.py

```diff
@@ -141,54 +176,74 @@ def convert_checkpoints(hf_model):
             hf_wts[f"h.{i}.mlp.dense_4h_to_h.bias"]
         )
 
-    # Save the model.
-    print(f"\n-> Saving KerasNLP model weights to `{FLAGS.preset}.weights.h5`.")
-    keras_model.save_weights(f"{FLAGS.preset}.weights.h5")
 
-    return keras_model
+def validate_output(
+    hf_model,
+    keras_model,
+    hf_tokenizer,
+    keras_tokenizer,
+):
+    print("⏺ checking Tokenizer and Backbone outputs")
```

Comment:
We can probably ditch this print, we are already printing at the end of each "stage"